public/documentation/getting_started/remote_debugging.md

```diff
@@ -182,13 +194,17 @@ Once you have the pre-requisites in place, turn on debugging for the Crosswalk s
       --external-extensions-path=/usr/lib/tizen-extensions-crosswalk
     ```
 
-3.  Reboot the Tizen target device.
+    
 
-Once the Crosswalk service is enabled for debugging, debug your applications as follows:
+3. Reboot the Tizen target device
 
-1.  Launch a Crosswalk application on the Tizen target using a console. See [these instructions](/documentation/getting_started/run_on_tizen.html#Run-the-application).
 
-2.  Back on the host, open a Chrome browser and open the address `http://&lt;Tizen target IP&gt;:9222`. For example, for the IP address 192.168.0.19, the URL to use would be "http://192.168.0.19:9222".
+
+4. Now that the Crosswalk service has been enabled for debugging, debug your application(s) as follows:
+
+    (a) Launch a Crosswalk application on the Tizen target using a console. See [these instructions](/documentation/getting_started/run_on_tizen.html#Run-the-application).
+
+    (b) Back on the host, open a Chrome browser and open the address `http://&lt;Tizen target IP&gt;:9222`. For example, for the IP address 192.168.0.19, the URL to use would be "http://192.168.0.19:9222".
 
```

Comment:
Although not a bug you introduced, the &lt; and &gt; are not changed to < >  when converting markdown.  Just use "<" and ">"